web/src/components/Configuration/Configuration.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import Header from '../Home/Header/Header';
+import Configuration1 from './Configuration1';
+import Configuration2 from './Configuration2';
+import Configuration3 from './Configuration3';
+import Configuration4 from './Configuration4';
+import Slider from "react-slick";
+import "slick-carousel/slick/slick.css"; 
+import "slick-carousel/slick/slick-theme.css";
+import './Configuration.css'
+class Configuration extends Component{
+
+    render(){
+        const settings = {
+            dots: true,
+            infinite: true,
+            speed: 1500,
+            slidesToShow: 1,
+            slidesToScroll: 1
+          };
+        return(
+            <div>
+                <Header/>
+                <Slider {...settings}>
+                <Configuration1/>
+                <Configuration2/>
+                <Configuration3/>
+                <Configuration4/>
```

Comment:
These 4 components (Configuration1, Configuration2, Configuration3 and Configuration4) are similar and have duplicated code. It would be very nice to extract a single component called PatientConfiguration that receives an argument to determine the current patient.
Inside the PatientConfiguration you could extract determine that argument from props and use it in the requests you send towards the server.